docs/language/accounts.mdx

```diff
@@ -110,6 +203,44 @@ transaction(key: [UInt8]) {
     }
 }
 ```
+</Callout>
+
+
+#### Get Account Keys
+Keys that are added to an account can be retrieved using `get()` function, using the index of
+the key. Revoked keys are always returned,  but they have `isRevoked` field set to true.
+Returns `nil` if there is no key available at the given index. Keys can be retrieved from both
+`PublicAccout` and `AuthAccount`.
+```cadence
+transaction() {
```

Comment:
Just my OCD sorry: Maybe add some whitespace and linebreaks (after the end of a sentence), here an in the other sections, to make this easier to edit, e.g.:

```suggestion
#### Get Account Keys
Keys that are added to an account can be retrieved using `get()` function, using the index of
the key. 
Revoked keys are always returned,  but they have `isRevoked` field set to true.
Returns `nil` if there is no key available at the given index. Keys can be retrieved from both
`PublicAccout` and `AuthAccount`.

```cadence
transaction() {
```